### PR TITLE
Fix/issues

### PR DESCRIPTION
**Pull request by ChristianMLux**

### CODE REVIEW

**Comment by glissario** [comment]
several open issues in this fix-branch (propably no clean work):
- validation of Questions
- reload-issue

File: src/components/AskQuestion/AskQuestion.vue [hunk 1]
```diff
@@ -113,36 +139,80 @@ export default {
...
-      this.currentQuestion.questionDescription = "";
+        this.currentQuestion.questionTitle = "";
+        this.currentQuestion.questionDescription = "";
+      }
     },
+
+    countWords(text) {
+      let count = 0;
+      for (let i = 0; i < text.length; i++) {
+        if (" " === text.charAt(i)) {
```
Comment (robinboehm) [inline src/components/AskQuestion/AskQuestion.vue:203]: You could also do this with `text.split(" ").length;`

Still not checks for newline etc ... But I think it's good enough for now.

File: src/components/AskQuestion/AskQuestion.vue [hunk 2]
```diff
@@ -113,36 +139,80 @@ export default {
...
-      this.questionArray.push(questionToList);
-      // creates database entry with given questionToList
-      DataService.create(questionToList)
-        .then(() => {})
-        .catch((e) => {
-          console.error(e);
-        });
+      let validation = this.validateQuestion();
+      if (!validation) {
+        console.log("no validation");
```
Comment (robinboehm) [inline src/components/AskQuestion/AskQuestion.vue]: What does that console.log does here?

File: src/components/AskQuestion/AskQuestion.vue [hunk 3]
```diff
@@ -91,11 +114,14 @@ export default {
...
+      validCategory: true,
+      validation: "",
       currentQuestion: {
         questionTitle: "",
         questionDescription: "",
-        questionCategory: "Simons category",
+        questionCategory: "",
         questionIsDone: false,
         questionCreated_at: new Date(),
         questionAuthor: "randomAuthor",
```
Comment (robinboehm) [inline src/components/AskQuestion/AskQuestion.vue]: Is this okay? Maybe it should be empty? I don't know. Just looks strange ;)

**Review by robinboehm — CHANGES_REQUESTED** [review]
Please fix some of the noted issues.

For the Future: Please write a more explaining issue text. I was confused that is happening here.

File: src/components/AskQuestion/AskQuestion.vue (hunk 3 above)
Comment (ChristianMLux) [reply src/components/AskQuestion/AskQuestion.vue]: this should already been fixed in another pull request.

Comment (ChristianMLux) [reply src/components/AskQuestion/AskQuestion.vue]: Oke.. somehow it isn't.. i will fix it.

File: src/components/AskQuestion/AskQuestion.vue (hunk 2 above)
Comment (glissario) [reply src/components/AskQuestion/AskQuestion.vue]: deleted the console.log -> we only need the validation case anyway

**Pull request merged**